lib/src/ffmpeg/ffmpeg_command.dart

```diff
@@ -33,7 +33,7 @@ class FfmpegCommand {
   const FfmpegCommand({
     this.inputs = const [],
     this.args = const [],
-    required this.filterGraph,
+    this.filterGraph,
```

Comment:
What is the purpose of this command without a filter graph?

Reply:
To allow for the use of this library without the need of using a filter_complex as not all tasks require filters such as simple copying while changing the duration of video, muting video or changing audio to another inputs audio. <br>
This was referenced in #7 - as there is no reason to force the user to have to use filters if in one case they require them but in another they do not, especially while in the same project which will require them to change to another library or install another library alongside this one - further increasing dependencies for no reason.

Reply:
Ok. In general, I prefer not to use magic `null` values to configure different behaviors. Given that the FFMPEG CLI, itself, has an explicit parameter to signify a "complex" run with a filter graph, how about we do the following?

```dart
const FfmpegCommand.simple({
 //...
});

const FfmpegCommand.complex({
 //...
});
```

Reply:
Mentioning that now, I do agree but never thought of doing it like that. <br>
I like the idea of keeping in line with the ffmpeg cli, so I'll add that in one of the newest pushes to the fork.